addons/project_todo/views/project_task_views.xml

```diff
@@ -80,7 +80,7 @@
                   default_group_by="personal_stage_type_ids"
                   editable="bottom"
                   multi_edit="1"
-                  js_class="todo_list">
+                  js_class="todo_list" open_form_view="True">
```

Comment:
Nitpicking to be consistent with the others attributes. 

```suggestion
                  open_form_view="True"
                  js_class="todo_list">
```

Reply:
changes done:slightly_smiling_face:  @xavierbol